### PR TITLE
fix: Adjust overview widget on dashboard for new design

### PR DESCRIPTION
**Pull request by usame-algan**
## What it solves

Updates the dashboard overview widget to support easy on-ramping

Figma: https://www.figma.com/file/ptTs6lDBeUuLNySroJ5PiF/Web-Master-File?node-id=8786%3A68251&mode=dev

## How to test it

1. Open a Safe
2. Go to the dashboard
3. Observe the updated design
4. Switch to a network where the Ramp Network app is activated
5. Observe a "Buy crypto" button

## Screenshots
Screenshot 2023-11-09 at 13 57 10
Screenshot 2023-11-09 at 13 57 27

## Checklist
* [ ] I've tested the branch on mobile 📱
* [ ] I've documented how it affects the analytics (if at all) 📊
* [ ] I've written a unit/e2e test for it (if applicable) 🧑‍💻

### CODE REVIEW

**Review by schmanu — APPROVED** [review]
LGTM!

File: public/images/common/arrow-se.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="21" height="20" viewBox="0 0 21 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M9.52621 12.4915H11.6919C12.1545 12.4915 12.5295 12.8665 12.5295 13.3291C12.5295 13.7916 12.1545 14.1666 11.6919 14.1666L7.50421 14.1666C7.27293 14.1666 7.06354 14.0729 6.91198 13.9213C6.76041 13.7698 6.66667 13.5604 6.66667 13.3291V9.14137C6.66667 8.67881 7.04165 8.30383 7.50421 8.30383C7.96677 8.30383 8.34175 8.67881 8.34175 9.14137V11.3071L13.5702 6.0786C13.8973 5.75152 14.4276 5.75152 14.7547 6.0786C15.0818 6.40568 15.0818 6.93599 14.7547 7.26307L9.52621 12.4915Z" fill="#121312"/>
```
Comment (schmanu) [inline public/images/common/arrow-se.svg:2]:
```suggestion
<path fill-rule="evenodd" clip-rule="evenodd" d="M9.52621 12.4915H11.6919C12.1545 12.4915 12.5295 12.8665 12.5295 13.3291C12.5295 13.7916 12.1545 14.1666 11.6919 14.1666L7.50421 14.1666C7.27293 14.1666 7.06354 14.0729 6.91198 13.9213C6.76041 13.7698 6.66667 13.5604 6.66667 13.3291V9.14137C6.66667 8.67881 7.04165 8.30383 7.50421 8.30383C7.96677 8.30383 8.34175 8.67881 8.34175 9.14137V11.3071L13.5702 6.0786C13.8973 5.75152 14.4276 5.75152 14.7547 6.0786C15.0818 6.40568 15.0818 6.93599 14.7547 7.26307L9.52621 12.4915Z" fill="currentColor"/>
```

File: public/images/common/arrow-top-right.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg width="21" height="20" viewBox="0 0 21 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M12.2355 6.84341H9.63669C9.08161 6.84341 8.63164 6.39344 8.63164 5.83836C8.63164 5.28329 9.08161 4.83331 9.63669 4.83331H14.6619C14.9395 4.83331 15.1907 4.94581 15.3726 5.12769C15.5545 5.30956 15.667 5.56083 15.667 5.83836V10.8636C15.667 11.4187 15.217 11.8687 14.6619 11.8687C14.1069 11.8687 13.6569 11.4187 13.6569 10.8636V8.26477L7.38272 14.5389C6.99022 14.9314 6.35386 14.9314 5.96136 14.5389C5.56887 14.1464 5.56887 13.5101 5.96136 13.1176L12.2355 6.84341Z" fill="#121312"/>
```
Comment (schmanu) [inline public/images/common/arrow-top-right.svg:2]:
```suggestion
<path fill-rule="evenodd" clip-rule="evenodd" d="M12.2355 6.84341H9.63669C9.08161 6.84341 8.63164 6.39344 8.63164 5.83836C8.63164 5.28329 9.08161 4.83331 9.63669 4.83331H14.6619C14.9395 4.83331 15.1907 4.94581 15.3726 5.12769C15.5545 5.30956 15.667 5.56083 15.667 5.83836V10.8636C15.667 11.4187 15.217 11.8687 14.6619 11.8687C14.1069 11.8687 13.6569 11.4187 13.6569 10.8636V8.26477L7.38272 14.5389C6.99022 14.9314 6.35386 14.9314 5.96136 14.5389C5.56887 14.1464 5.56887 13.5101 5.96136 13.1176L12.2355 6.84341Z" fill="currentColor"/>
```

**Review by schmanu — CHANGES_REQUESTED** [review]
I just realized that in Dark mode the icons do not adjust their color.

**Review by schmanu — APPROVED** [review]
The app is also listed now on staging and the button works :)

**Comment by katspaugh** [comment]
The skeletons need some adjustments, it jumps twice when loading.

https://github.com/safe-global/safe-wallet-web/assets/381895/1d2d3242-14a1-498e-908f-89de91d03eec

**Comment by francovenica** [comment]
For mobile if the "Buy" button is present is all too cramped:
image

Question just out of curiosity: why the "Buy" is an "a" tag, the "Send" is a button and the "Receive" is a button wrapped in a div?

**Comment by usame-algan** [comment]
I updated the loading state and mobile view accordingly

> Question just out of curiosity: why the "Buy" is an "a" tag, the "Send" is a button and the "Receive" is a button wrapped in a div?

The Buy Crypto button is an `a` because it links to a different page. The Receive button is wrapped in a `div` because we are reusing a wrapper component that adds an onClick handler around its children so there has to be some element to hold that.

**Comment by katspaugh** [comment]
Can you make Send and Receive be on the same line, and Buy crypto on a different line?

**Comment by usame-algan** [comment]
Changed it like so on mobile:
Screenshot 2023-11-10 at 11 40 57

**Pull request merged**